pkg/provider/browser/browser_test.go

```diff
@@ -102,7 +102,7 @@ func TestNoBrowserDriverFail(t *testing.T) {
 	client, _ := New(account)
 	_, err := client.Authenticate(loginDetails)
 	assert.Error(t, err)
-	assert.ErrorContains(t, err, "could not start driver")
```

Comment:
Thank you for this change @charlottecampbell193 - is there a scenario where the error "could not start driver" will be thrown? Or has that been completely removed?

Reply:
@mapkon that error can still be thrown but not with the current test. A new test would need to be created that has the correct driver installed but has some other issue that is preventing it from starting. I'm not sure what scenario that would be though.

Reply:
You are right since the current error message is probably faulty, and `playwright` has fixed their API. The current test was exzercising the scenario where there is completely no driver, so it makes sense to throw the `please install the driver` message. The comment on the test gave it away:
`// Test that if download directory does not have browsers, it fails with expected error message`


I imagine that we should have a test that launches a directory with an existing driver that we know won't "start", and see if that will fire it up (probably an invalid browser binary or similar). I wouldn't spend more than a day on it if you cannot get it to start.

Lastly, remember to update the README on line 575 to reflect the new message that will get thrown if the user attempts use without installing the drivers (like on first time use).

Reply:
I was not able to recreate that error message. When I had an invalid browser binary the error I got was `'playwright: spawn UNKNOWN'`. Tried a bunch of stuff messing around with it and got many other unhelpful messages but not `could not start driver`.

README has been updated - let me know your thoughts on the update.

Reply:
Suffices for me. We can merge it now or wait for #1302 to be fixed (so we can run all the tests). Let me know